tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./app/templates/*.html", "./app/templates/auth/*.html"],
```

Comment:
Also could be convenient to use something like "./templates/**/*.{html}"

Reply:
thank you. fixed